addons/survey/models/survey_user.py

```diff
@@ -50,13 +50,16 @@ class SurveyUserInput(models.Model):
         ('unique_token', 'UNIQUE (access_token)', 'An access token must be unique!'),
     ]
 
-    @api.depends('user_input_line_ids.answer_score', 'user_input_line_ids.question_id')
+    @api.depends('user_input_line_ids.answer_score', 'user_input_line_ids.question_id', 'user_input_line_ids.answer_type')
```

Comment:
Seems like `user_input.predefined_question_ids.answer_score` would be more appropriated.

Don't see why you added `user_input_line_ids.answer_type`.

Reply:
I think I added `user_input_line_ids.answer_type` by mistake or it was leftover from an older version of how I wrote the logic. I don't really remember anymore to be honest. Either way, fixed it.